content/launch_2-firmware-update.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Manual steps to update firmware on Launch and Launch Heavy keyboards
+description: >
+  Launch firmware before version 0.12.20 will need these manual steps to make sure they can reliably upgrade firmware with `fwupd`
+keywords:
+  - Firmware
+  - Open Firmware
+  - fwupd
+  - Launch_2
+  - Launch_heavy_1
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software-troubleshooting
+tableOfContents: true
+---
+
+## Steps to take to update firmware
+
+* NOTE: This should only be necessary for firmware versions starting with `0.7.103-`. Newer versions should be able to update via the Firmware Manager in Pop!\_OS and Ubuntu, or via `fwupdmgr`.
+
+If you are receiving notifications that your `launch_2` or `launch_heavy_1` keyboard has an available firmware update, but the update won't install, this guide may help you manually flash the firmware update. We will be using the [System76 Keyboard Configurator](https://github.com/pop-os/keyboard-configurator/), but it will be in the testing mode that we use for factory firmware updates, as well as quality control.
+
+Manually updating firmware in this way will reset your Launch back to its factory defaults, undoing any customizations that you've made to mapping and LEDs. Before you begin, you may want to use the System76 Keyboard Configurator to export your layout. After the firmware update, it can be imported to restore all of your customizations.
+
+![Exporting Layout in Keyboard Configurator](/images/launch_2-firmware-update/heavy-export-layout.png)
+
+### Install software
+
+We have a package that will launch the System76 Keyboard Configurator in testing mode, which can be used to install this firmware update. Run the following command in a terminal:
+
+```
+sudo apt install test-launch-system76-keyboard-configurator
+```
+
+If you are not on Pop!\_OS or Ubuntu, you can also use the AppImage version of the System76 Keyboard Configurator and launch it with the `--launch-test` command line argument. You will also need to install `avrdude` and `dfu-programmer` if they are not installed already.
+
+Start System76 Keyboard Tester -- this will launch the System76 Keyboard Configurator in a test mode.
+
+![System76-Tester](/images/launch_2-firmware-update/Launch-system76-keyboard-tester.png)
+
+### Enter Bootloader mode
+
+Unplug the launch keyboard and hold <kbd>Esc</kbd> while plugging it back in. Continue to hold the <kbd>Esc</kbd> key for about 10 seconds, or until the firmware update button appears in the right side of the Configurator's header bar. The keyboard's LEDs will remain off while the keyboard is in bootloader mode.
```

Comment:
I believe in "Unplug the launch keyboard" Launch Keyboard should be capitalized.

Reply:
Fixed with [0c840a2](https://github.com/system76/docs/pull/1140/commits/0c840a2689c4bb874b121306f7241aa6c1d48363)